ubg/src/Reviews/NewReview.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+class NewReview extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            rating: '',
+            text: '',
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.addReview = this.addReview.bind(this);
+    }
+
+    handleChange(event) {
```

Comment:
Do you need this function? You are not using it

Reply:
Deleted the function